src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,180 @@
+package duke;
+
+import java.io.File;
+import java.util.Scanner;
+
+import javafx.application.Application;
```

Comment:
I would prefer to arrange the imports as such

package duke;

import javafx.application.Application;
import javafx.scene.Scene;
import javafx.scene.control.Button;
import javafx.scene.control.Label;
import javafx.scene.control.ScrollPane;
import javafx.scene.control.TextField;
import javafx.scene.image.Image;
import javafx.scene.image.ImageView;
import javafx.scene.layout.AnchorPane;
import javafx.scene.layout.Region;
import javafx.scene.layout.VBox;
import javafx.stage.Stage;

import java.io.File;
import java.util.Scanner;